crates/stc_ts_errors/src/lib.rs

```diff
@@ -201,6 +201,10 @@ pub enum Error {
         span: Span,
     },
 
+    TupleRestElementMustBeArrayType {
+        span: Span,
+    },
+
```

Comment:
I couldn't see any tests in the [TypeScript](https://github.com/microsoft/typescript) repo for this